### ISSUE TITLE
Could it be possible to train DETR from scratch on COCO?

### ISSUE THREAD

**Issue by JrPeng** [facebookresearch/detr#157]
If possible, how long it may take to converge to a state like finetuning for 300 epochs?
Are there some benchmarks about DETR with various types of backbones?
Thank you very much.

**Comment by szagoruyko** [comment]
@JrPeng Hi, we didn't try many things to train from scratch. The best result we got is 33 AP (300 epochs) with the following modifications:
* use R50 with group norm
* use pre-norm in Transformer
* set the same learning rate of 1e-4 to both ResNet and Transformer

> Are there some benchmarks about DETR with various types of backbones?

no I don't think so. There are R50 results for SwAV and Moco-v2 in https://arxiv.org/abs/2006.09882, which improve AP at 300 epochs quite significantly, more than 1 AP.

**Comment by JrPeng** [comment]
Thank you, no more questions at the moment.
